dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -27,9 +28,16 @@ const JobDetailPage = () => {
   const jobId = params.id;
   const { progress } = useJobProgress(jobId);
 
+  const pageInfo = {
```

Comment:
Could we use the value directly inside `pageInfo={}`? Seems like the conditional is evaluated below on L37